support/cas-server-support-inwebo-mfa/src/main/java/org/apereo/cas/support/inwebo/web/flow/actions/CheckAuthenticationAction.java

```diff
@@ -0,0 +1,72 @@
+package org.apereo.cas.support.inwebo.web.flow.actions;
+
+import org.apereo.cas.support.inwebo.authentication.InweboCredential;
```

Comment:
Please make sure all webflow action classes are prefixes with Inwebo, similar to simple-mfa.

Reply:
I think I already answered this one, but it's done.